lib/debug/thread_client.rb

```diff
@@ -1187,8 +1188,11 @@ def enabled?
         @tp_recorder.enabled?
       end
 
-      def step_back
-        @index += 1
+      def step_back iter
+        @index += iter
+        if @index > @log.size
+          @index = @log.size
```

Comment:
When n of "step  back <n>" is larger than current `@index`, `@index` will be size of log.